### PR TITLE
Fix #22: Use csslint-stylish as the default formatter

### LINKED ISSUE #22: use a recommented reporter.
There is no need for us to have a custom reporter, gulp-csslint can use differnt reporters.
the one recommended for this issue is [csslint-stylish](https://www.npmjs.com/package/csslint-stylish).

### PR DESCRIPTION
**Pull request by paul-barry-kenzan**
**Associated Ticket**
#22 

**Changes Made**
  - Remove standalone formatter
  - Utilize `csslint-stylish` as the default output formatter

**Reviewers**
@thescientist13 
@cxoltero 
@bsawyer

### CODE REVIEW

**Comment by thescientist13** [comment]
Can one of the admins verify this patch?

**Comment by thescientist13** [comment]
ok to test

**Comment by thescientist13** [comment]
@paul-barry-kenzan 
Should this be rebased / reviewed after #25 ?

**Comment by paul-barry-kenzan** [comment]
@thescientist13 
Yes it'll require a rebase. It's stems off of the branch from #25

**Comment by thescientist13** [comment]
@paul-barry-kenzan 
Please rebase my good man

**Comment by paul-barry-kenzan** [comment]
@thescientist13 
Rebased and ready to be merged.

File: src/index.js [hunk 1]
```diff
@@ -21,7 +20,7 @@ function pipelineFactory (config) {
 
   stream = lazypipe()
     .pipe(cssLint, config)
-    .pipe(cssLint.formatter, formatter);
+    .pipe(cssLint.formatter, require('csslint-stylish'));
```
Comment (thescientist13) [inline src/index.js]: keeping with convention, I think it might make sense to keep this `require` at the top, and pass the reference in.  Just a thought.

Comment (paul-barry-kenzan) [inline src/index.js]: @thescientist13 
Yeah, it's best to declare it at the top. I'll get that change in.

**Pull request merged**